graph_module/EGraph graph
geometry_msgs/Point centroid
